File: combo-combat-unity/Assets/Networking Scripts/NetworkLevelLoad.js
function Awake ()
{
	// Network level loading is done in a seperate channel.
	//DontDestroyOnLoad(this);
}

function OnConnectedToServer() {
    Network.SetSendingEnabled(0, false);	
    Network.isMessageQueueRunning = false;
    Application.LoadLevel("ComboGame");
}

function OnServerInitialized() {
    Application.LoadLevel("ComboGame");
    Debug.Log("Server initialized and ready");
}
function OnDisconnectedFromServer ()
{
    //Application.LoadLevel(disconnectedLevel);
    // TODO: Reconnecting...
}

@script RequireComponent(NetworkView)

File: combo-combat-unity/Assets/Scripts/Networking/NetworkLevelLoad.cs
using UnityEngine;
using System.Collections;

[RequireComponent(typeof(NetworkView))]
public class NetworkLevelLoad : MonoBehaviour {

    void OnConnectedToServer() {
        Network.SetSendingEnabled(0, false);
        Network.isMessageQueueRunning = false;
        LoadArenaLevel();
    }

    void OnServerInitialized() {
        LoadArenaLevel();
        Debug.Log("Server initialized and ready");
    }

    void LoadArenaLevel() {
        Network.SetLevelPrefix(GameConstants.LEVEL_PREFIX_MATCH);
        Application.LoadLevel(GameConstants.LEVEL_MATCH);
    }

}